openbb_sdk/sdk/core/openbb_core/api/router/commands.py

```diff
@@ -166,6 +166,6 @@ def add_command_map(command_runner: CommandRunner, api_router: APIRouter) -> Non
     api_router.include_router(router=plugins_router.api_router)
 
 
-system_settings = SystemService.read_default_system_settings()
+system_settings = SystemService(**{"logging_sub_app": "api"}).system_settings
```

Comment:
```suggestion
system_settings = SystemService(logging_sub_app="api").system_settings
```

Reply:
Thank you, not sure why I've typed it like that :laughing: